eng/pipelines/mgmt.yml

```diff
@@ -1,25 +1,163 @@
+    - sdk/cognitiveservices/FormRecognizer
+    - sdk/cognitiveservices/Knowledge.QnAMaker
+    - sdk/cognitiveservices/Language.LUIS.Authoring
+    - sdk/cognitiveservices/Language.LUIS.Runtime
+    - sdk/cognitiveservices/Language.SpellCheck
+    - sdk/cognitiveservices/Language.TextAnalytics
+    - sdk/cognitiveservices/Personalizer
+    - sdk/cognitiveservices/Search.BingAutoSuggest
+    - sdk/cognitiveservices/Search.BingCustomImageSearch
+    - sdk/cognitiveservices/Search.BingCustomSearch
+    - sdk/cognitiveservices/Search.BingEntitySearch
+    - sdk/cognitiveservices/Search.BingImageSearch
+    - sdk/cognitiveservices/Search.BingLocalSearch
+    - sdk/cognitiveservices/Search.BingNewsSearch
+    - sdk/cognitiveservices/Search.BingVideoSearch
+    - sdk/cognitiveservices/Search.BingVisualSearch
+    - sdk/cognitiveservices/Search.BingWebSearch
+    - sdk/cognitiveservices/Vision.ComputerVision
+    - sdk/cognitiveservices/Vision.ContentModerator
+    - sdk/cognitiveservices/Vision.CustomVision.Prediction
+    - sdk/cognitiveservices/Vision.CustomVision.Training
+    - sdk/cognitiveservices/Vision.Face
+    - sdk/containerregistry/Microsoft.Azure.ContainerRegistry
+    - sdk/core/Azure.Core
+    - sdk/core/Microsoft.Extensions.Azure
+    - sdk/eventgrid/Microsoft.Azure.EventGrid
+    - sdk/eventhub/Azure.Messaging.EventHubs
+    - sdk/eventhub/Azure.Messaging.EventHubs.Processor
+    - sdk/eventhub/Azure.Messaging.EventHubs.Shared
+    - sdk/eventhub/Microsoft.Azure.EventHubs
+    - sdk/eventhub/Microsoft.Azure.EventHubs.Processor
+    - sdk/eventhub/Microsoft.Azure.EventHubs.ServiceFabricProcessor
+    - sdk/graphrbac/Microsoft.Azure.Graph.RBAC
+    - sdk/hdinsight/Microsoft.Azure.HDInsight.Job
+    - sdk/identity/Azure.Identity
+    - sdk/keyvault/Azure.Security.KeyVault.Certificates
+    - sdk/keyvault/Azure.Security.KeyVault.Keys
+    - sdk/keyvault/Azure.Security.KeyVault.Secrets
+    - sdk/keyvault/Azure.Security.KeyVault.Shared
+    - sdk/keyvault/Microsoft.Azure.KeyVault
+    - sdk/keyvault/Microsoft.Azure.KeyVault.Core
+    - sdk/keyvault/Microsoft.Azure.KeyVault.Cryptography
+    - sdk/keyvault/Microsoft.Azure.KeyVault.Extensions
+    - sdk/keyvault/Microsoft.Azure.KeyVault.WebKey
+    - sdk/operationalinsights/Microsoft.Azure.OperationalInsights
+    - sdk/search/Microsoft.Azure.Search
+    - sdk/search/Microsoft.Azure.Search.Common
+    - sdk/search/Microsoft.Azure.Search.Data
+    - sdk/search/Microsoft.Azure.Search.Service
+    - sdk/servicebus/Microsoft.Azure.ServiceBus
+    - sdk/storage/Azure.Storage.Blobs
+    - sdk/storage/Azure.Storage.Blobs.Batch
+    - sdk/storage/Azure.Storage.Blobs.Cryptography
+    - sdk/storage/Azure.Storage.Common
+    - sdk/storage/Azure.Storage.Files.DataLake
+    - sdk/storage/Azure.Storage.Files.Shares
+    - sdk/storage/Azure.Storage.Queues
+    - sdk/template/Azure.Template
+    - sdk/textanalytics/Azure.AI.TextAnalytics
+pr:
+  branches:
+    include:
+    - master
+    - '*-preview'
```

Comment:
What is this branch pattern for?

Reply:
That's for public review branches

Reply:
Are those branches in the main repository? How much work would it be to make those branch names conform to the branching naming guidelines at https://github.com/Azure/azure-sdk/blob/master/docs/policies/repobranching.md

Reply:
Yes. There are currently 4 such branches, all stale. I believe it's safe and easy to rename them